front/components/customField/InstitutionsDialog.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <v-dialog
+    v-model="isOpen"
+    :width="loading ? 200 : 900"
+    scrollable
+  >
+    <LoaderCard v-if="loading" />
+
+    <v-card
+      v-else
+      :title="$t('repositories.institutions')"
```

Comment:
Show count of institutions
```suggestion
      :title="$t('institutions.toolbarTitle', { count: fieldData.length })"
```